.github/workflows/build.yaml

```diff
@@ -584,6 +584,60 @@ jobs:
           name: Manpages
           path: ${{env.RouterBuildDir}}/docs/man/*.html
 
+  rpm:
+    name: 'Build and test RPM (${{ matrix.container }}:${{ matrix.containerTag }})'
+    runs-on: '${{ matrix.os }}'
+    strategy:
+      matrix:
+        os: [ 'ubuntu-20.04' ]
+        container: [ 'centos' ]
+        containerTag: [ 'stream8' ]
+
+    container:
+      image: 'quay.io/${{ matrix.container }}/${{ matrix.container }}:${{ matrix.containerTag }}'
+      volumes:
+        - ${{github.workspace}}:${{github.workspace}}
+
+    env:
+      DNF: ${{github.workspace}}/build
+
+    steps:
+      - name: Install build dependencies (1/2)
+        run: |
+          dnf config-manager --set-enabled powertools
+          dnf install --setopt=tsflags=nodocs --setopt=install_weak_deps=False -y epel-release 'dnf-command(copr)' 'dnf-command(builddep)'
+          dnf copr enable -y clime/rpkg-util
+          dnf install --setopt=tsflags=nodocs --setopt=install_weak_deps=False -y git rpkg
+
+      - uses: actions/checkout@v2
+
+      - name: Install build dependencies (2/2)
+        run: |
+          dnf install --setopt=tsflags=nodocs --setopt=install_weak_deps=False -y gcc gcc-c++
+          rpkg spec --outdir /tmp/rpkg
+          dnf builddep --setopt=tsflags=nodocs --setopt=install_weak_deps=False -y /tmp/rpkg/skupper-router.spec
+
+      - name: Build packages
+        run: |
+          mkdir /tmp/skupper-rpms
+          rpkg local --nocheck --outdir /tmp/skupper-rpms
+
+      - name: Install built packages
+        run: |
+          dnf install -y /tmp/skupper-rpms/*/*.rpm
+
+      - name: Check that skrouterd works
+        run: |
+          skrouterd -c /dev/empty |& grep "Configuration file could not be opened"
```

Comment:
Ideally, this should test each package independently (Does -server package work without -tools being installed?) and more in depth. Sadly, the -tests package is actually broken and the system tests don't run against installed skrouterd (this is known ENTMQIC issue). For now, this should do. It checks that Python internal module can be loaded, because that happens before router accesses config.

Reply:
Can you open a corresponding GH skupper-router issue for this -tests package breakage?

Reply:
https://github.com/skupperproject/skupper-router/issues/267